cumulusci/tasks/metadata/package.py

```diff
@@ -362,6 +362,30 @@ def _parse_item(self, item):
         return [item]
 
 
+class LightningComponentBundleParser(BaseMetadataParser):
+    def _parse_item(self, item):
+        members = []
+        path = self.directory + "/" + item
+
+        # Skip non-directories
+        if not os.path.isdir(path):
+            return members
+
+        # Add the member if it is not namespaced
+        if "__" not in item:
+            members.append(item)
+
+        # ignore subitems
+
+        return members
+
+    def check_delete_excludes(self, item):
+        return False
+
+    def _parse_subitem(self, item, subitem):
+        return []
```

Comment:
What do you think about replacing both `AuraBundleParser` and `LightningComponentBundleParser` with a single `BundleParser` that is used for both?

Skipping non-directories won't cause problems for Aura.

I don't think we need the check for namespaced members. I assume a package can't override an LWC from another namespace.

I think you can leave out `check_delete_excludes` (the inherited implementation looks innocuous and could be handy sometime) and `_parse_subitem` (it's only called from BaseMetadataParser's `_parse_item` which you've replaced)

Reply:
I will make the changes

Reply:
Changes are commited and pushed